[MA-902] clarify no_data_timeframe requirements/recommendations (#6737)
DataDog/documentation: content/en/api/monitors/monitors_create.md

    *   **`notify_no_data`** a Boolean indicating whether this monitor notifies when data stops reporting. Default: **false**

>     *   **`no_data_timeframe`** The number of minutes before a monitor will notify when data stops reporting.  We recommend at least 2x the monitor timeframe for metric alerts or 2 minutes for service checks.  **If omitted, we fall back to the evaluation timeframe for metric alerts, and 24 hours for service checks.**

Reviewer: ```suggestion
    *   **`no_data_timeframe`** The number of minutes before a monitor notifies after data stops reporting. Datadog recommends at least 2x the monitor timeframe for metric alerts or 2 minutes for service checks.  **If omitted, the evaluation timeframe is used for metric alerts, and 24 hours is used for service checks.**
```
